src/crawlee/events/_local_event_manager.py

```diff
@@ -32,8 +33,12 @@ def __init__(
     ) -> None:
         """A default constructor.
 
+        In the most cases you should use the `from_config` constructor to create an new instance based on
+        the provided configuration.
+
         Args:
-            system_info_interval: Interval at which `SystemInfo` events are emitted.
+            system_info_interval: Interval at which `SystemInfo` events are emitted. The default value is taken
+                from the `Configuration`.
```

Comment:
Weird thing to say here - the constructor does not receive the `Configuration` object even if `from_config` is used.

Reply:
This is a forgotten docstring update from the previous version. Thanks.